tests/pages/course-page.js

```diff
@@ -149,7 +153,14 @@ export default create({
   privateLeaderboardFeatureSuggestion: PrivateLeaderboardFeatureSuggestion,
 
   progressBannerModal: {
+    clickOnCopyButton: clickable('[data-test-copy-button]'),
+
+    copyableText: {
+      scope: '[data-test-copyable-text]',
+    },
+
     scope: '[data-test-progress-banner-modal]',
```

Comment:
@libmartinito this modal is different. It's for the github progress banner. The events we're adding here are for ShareProgressModal